spec/lib/manageiq/api/common/paginated_response_spec.rb

```diff
@@ -0,0 +1,66 @@
+describe ManageIQ::API::Common::PaginatedResponse do
+  context "private links methods" do
+    let(:base_query) { double("AR:Clause", :count => count) }
+    let(:request) { double("Request", :original_url => "http://example.com/resource?param1=true&limit=#{limit}") }
+
+    def url_with_offset(offset)
+      "http://example.com/resource?limit=#{limit}&offset=#{offset}&param1=true"
+    end
+
+    context "with 6 records, limit 2" do
```

Comment:
The name of the context shouldn't describe what's in it, but instead should describe what the purpose of the context is, IMO.